homeassistant/components/sensibo/climate.py

```diff
@@ -146,6 +147,9 @@ def __init__(
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, coordinator.data[device_id]["id"])},
             name=coordinator.data[device_id]["name"],
+            connections={
+                (CONNECTION_NETWORK_MAC, format_mac(coordinator.data[device_id]["mac"]))
```

Comment:
Its not needed to normalize these here, as the DeviceRegistry will already take care of that.

```suggestion
                (CONNECTION_NETWORK_MAC, coordinator.data[device_id]["mac"])
```